### DISCUSSION TITLE
Module federation requesting remote component from wrong origin

### DISCUSSION THREAD

**Discussion by leolima** [webpack/webpack#18829] [Q&A]
I'm using Module federation plugin for nextjs.

I have 2 applications. (host and calc) When I try access the calc (http://localhost:3001) on browser, using a remote component from host (http://localhost:3000/). The browser is requesting the source from wrong origin.

![image](https://github.com/user-attachments/assets/5885e742-39c0-4293-8213-a64372287d72)

![image](https://github.com/user-attachments/assets/e68bcb29-f0f1-46ad-8b85-10c0c0eb0000)

check out my 'nextjs.config.js'

    const { NextFederationPlugin } = require('@module-federation/nextjs-mf');
    const path = require('path');
    
    module.exports = {
      basePath: '/calc',
      webpack(config, options) {
        if (!options.isServer) {
          config.plugins.push(
            new NextFederationPlugin({
              name: 'calc',
              filename: 'static/chunks/remoteEntry.js',
              remotes: {
                host: `host@http://localhost:3000/v2/_next/static/chunks/remoteEntry.js`,
              },
              exposes: {},
              shared: {},
              publicpath: 'auto',
            }),
          );
        }
    
        return config;
      },
    };

**Comment by alexander-akait** [comment]
please ask this in `nextjs` repo

**Reply by leolima** [reply]
Sorry, you are right.
